documentation/configuration.md

```diff
@@ -213,10 +213,19 @@ It is possible to change this behavior and have .NET processes connect to `dotne
 </details>
 
 <details>
-  <summary>Environment Variables</summary>
+  <summary>ConfigMap</summary>
   
   ```bash
-  export DotnetMonitor_DiagnosticPort="\\\\.\\pipe\\dotnet-monitor-pipe"
+  DotnetMonitor_DiagnosticPort: "\\\\.\\pipe\\dotnet-monitor-pipe"
+  ```
+</details>
+
+<details>
+  <summary>Kubernetes Environment Variables</summary>
+  
+  ```bash
+  - name: DotnetMonitor_DiagnosticPort
+    value: "\\\\.\\pipe\\dotnet-monitor-pipe"
```

Comment:
Consider changing this example since pipe won't make much sense in an Aks/ConfigMap scenario

Reply:
I would change it to `/diag/port.sock` or something similar.

Reply:
I put `/diag/port.sock` for the two Kubernetes formats, and left the JSON value untouched - if we want the JSON one to also be `/diag/port.sock`, just let me know and I can switch that too, but I figured it made sense to keep the pipe there.

Reply:
I would make them all of the same so that anyone who reads it knows that they are the same property being set.